modules/mesaides/files/ma-web.conf

```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
+    set -a
+    if [ -f /home/main/.secrets ]
+    then
+         . /home/main/.secrets
```

Comment:
Prefer `source` over `.`, they are [synonymous](https://askubuntu.com/questions/25488/what-is-the-difference-between-source-and-in-bash) but `source` is muuuuuch more readable.